tests/statespace/models/structural/components/test_level_trend.py

```diff
@@ -209,6 +217,7 @@ def test_add_level_trend_with_different_observed():
     )
 
 
+@pytest.mark.filterwarnings("ignore:Duplicate names found:UserWarning")
```

Comment:
duplicates shouldn't ever warn, it should raise (if disallowed) or silently accept it.